src/test/java/com/booster/server/NoteControllerTest.java

```diff
@@ -18,30 +24,71 @@
 @AutoConfigureMockMvc
 class NoteControllerTest {
 
+    final String firstNote = "My first note";
+    final String secondNote = "Read 10 pages";
+    final String thirdNote = "Practice piano";
+
     @Autowired
     MockMvc mockMvc;
     @Autowired
     ObjectMapper objectMapper;
     @Autowired
     NoteRepository noteRepository;
 
+    @BeforeEach
+    void beforeEach() {
+        noteRepository.deleteAll();
+    }
+
     @Test
     void createNewNote() throws Exception {
         assertThat(noteRepository.findAll()).isEmpty();
 
-        String content = "My first note";
-
         mockMvc.perform(MockMvcRequestBuilders.post("/note")
                         .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .content(objectMapper.writeValueAsString(new CreateNoteInput().setContent(content))))
+                        .content(objectMapper.writeValueAsString(new CreateNoteInput().setContent(firstNote))))
                 .andDo(print())
                 .andExpect(status().isCreated())
-                .andExpect(jsonPath("$.content").value(content));
+                .andExpect(jsonPath("$.content").value(firstNote));
 
         assertThat(noteRepository.findAll())
                 .hasSize(1)
                 .extracting("content")
-                .containsExactly(content);
+                .containsExactly(firstNote);
+    }
+
+    @Test
+    void correctUpdateOfLastSeenAtPropertyOfVocabularyEntry() throws Exception {
+        assertThat(noteRepository.findAll()).isEmpty();
+
+        LocalDateTime initDateTime = LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIN);
```

Comment:
If this is moved to the entity we can get rid of explicit `lastSeenAt` initialization.